src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -704,14 +730,24 @@ public uint256 LastReceivedBlockHash()
 
         /// <inheritdoc />
         public IEnumerable<UnspentOutputReference> GetSpendableTransactionsInWallet(string walletName, int confirmations = 0)
+        {
```

Comment:
Can't really tell the difference between `GetSpendableTransactionsInWallet` and  `GetSpendableTransactionsInWalletForStaking`.

If we call `GetSpendableTransactionsInWallet(walletName, conf)`, `GetSpendableTransactionsInWallet(walletName, confirmations, Wallet.NormalAccounts)` will be called, and if we call `GetSpendableTransactionsInWalletForStaking(walletName, conf)`, then it will call `GetSpendableTransactionsInWallet(walletName, conf)`, which will call `GetSpendableTransactionsInWallet(walletName, confirmations, Wallet.NormalAccounts)`.